setup.py

```diff
@@ -35,14 +35,35 @@
 with open(".dep-versions") as f:
     jax_version = [line[4:].strip() for line in f.readlines() if "jax=" in line][0]
 
+pl_version = environ.get("pl_version", ">=0.32,<=0.34")
```

Comment:
Let's use UPPERCASE env variables for the consistency:

```suggestion
pl_version = environ.get("PL_VERSION", ">=0.32,<=0.34")
```